AttributeCategoryForCAP/examples/example.g

```diff
@@ -82,44 +82,44 @@ structure_record.morphism_type := TheTypeOfMorphismsOfObjectsWithEndomorphism;;
 
 ## This is the mathematical core: provide the functions that enhance objects with attributes
 structure_record.ZeroObject :=
-  function( underlying_zero_object )
+  function( cat, underlying_zero_object )
```

Comment:
This is a backwards incompatible change, right? Does passing the category have a concrete advantage?

I agree that passing the category would be the "modern way" of doing things, but I would suggest not breaking the backwards compatibility without a concrete advantage. (Additionally, it looks like `cat` is the AttributeCategory but maybe the underlying category would make more sense.)

Reply:
My comment was meant to apply to all entries of `structure_record`. What is special about `ZeroObject` and `DualOnObjects` that you only reverted those two?